CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed [here ](https://github.com/sensu-plugin
 
 ## [Unreleased]
 
+### Changed
+
```

Comment:
no need for an extra newline here but no biggie I can clean that up for you post acceptance.